Index: gtk+-3.22.21/gdk/wayland/gdkglcontext-wayland.c
===================================================================
--- gtk+-3.22.21.orig/gdk/wayland/gdkglcontext-wayland.c
+++ gtk+-3.22.21/gdk/wayland/gdkglcontext-wayland.c
@@ -117,6 +117,8 @@ gdk_wayland_gl_context_realize (GdkGLCon
   gboolean debug_bit, forward_bit, legacy_bit, use_es;
   int i = 0;
 
+  _gdk_gl_flags = (GDK_GL_LEGACY | GDK_GL_GLES);
+
   gdk_gl_context_get_required_version (context, &major, &minor);
   debug_bit = gdk_gl_context_get_debug_enabled (context);
   forward_bit = gdk_gl_context_get_forward_compatible (context);
